src/export_balances.ts

```diff
@@ -547,7 +547,7 @@ async function main() {
   for (const [account, balanceBN] of balances) {
     const balance = balanceBN.toNumber()
     total += balance
-    if (balance > 0) {
+    if (balance > 10) {
```

Comment:
is that the existential deposit value amount ?

Reply:
yes, https://github.com/Joystream/joystream/blob/olympia/runtime/src/constants.rs#L265